questions/qa-bidi-unicode-controls.en.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<meta charset="utf-8" />
+<meta charset="utf-8">
 <title>How to use Unicode controls for bidi text</title>
-<meta name="description" content="If I'm unable to use markup to correctly order bidirectional text, what can I do?" />
+<meta name="description" content="If I'm unable to use markup to correctly order bidirectional text, what can I do?">
```

Comment:
Is there a reason to phrase this backwards?

```suggestion
<meta name="description" content="What can I do if I'm unable to use markup to correctly order bidirectional text?">
```